src/Motions/Word.ts

```diff
@@ -8,45 +8,56 @@ export class MotionWord extends Motion {
     private wordDelta: MotionWordPosition;
     private useBoundaryIfChange = false;
 
-    static wordSeparators = './\\\\()"\'\\-:,.;<>~!@#$%^&*|+=\\[\\]{}`~?';
+    private static wordSeparators = './\\\\()"\'\\-:,.;<>~!@#$%^&*|+=\\[\\]{}`~?';
+    private static blankSeparators = '\\s';
+    
+    separators: string;
 
-    static nextStartOrBoundaryIfChange(): Motion {
-        const obj = MotionWord.nextStart();
+    constructor(options: {useBlankSeparatedStyle: boolean} = {useBlankSeparatedStyle: false}) {
+        super();
+
+        this.separators = options.useBlankSeparatedStyle
+            ? MotionWord.blankSeparators : MotionWord.wordSeparators;
+    }
+
+    static nextStartOrBoundaryIfChange(args: {useBlankSeparatedStyle: boolean} = {useBlankSeparatedStyle: false}): Motion {
```

Comment:
Define default value like this may have some unexpected behavior.
Say we pass `args: {someOtherParam: someValue}` to the function, the `args.useBlankSeparatedStyle` will be `undefined`.

It's safer to define it like this:

```ts
static nextStartOrBoundaryIfChange(args: {useBlankSeparatedStyle: boolean} = {}): Motion {
    args.useBlankSeparatedStyle = args.useBlankSeparatedStyle === undefined ? false : args.useBlankSeparatedStyle;
    // ...
}
```